modules/allBook.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable import/prefer-default-export */
+export class AllBook {
```

Comment:
- [ ] Please avoid disabling linters. Linters are important for maintaining coding standards across projects. You fix this error like so:

  - Remove the linter disabling.
  - Change line 2 to this:
    
    ```
    export default class AllBook {
    ```
  - Then in the import in `src/index.js` file, you should change to this:
  
    ```
    import AllBook from './modules/allBook.js';
    ```